### ISSUE TITLE
Provide a way to run *all* tests.

### ISSUE THREAD

**Issue by vext01** [rust-lang/cargo#5419]
So we have:
```
cargo test
```

To run all but ignored tests. Then:
```
cargo test -- --ignored
```

To run only ignored tests.

But as far as I can see there is no way to run **all** tests in parallel short of:
```
cargo test & cargo test -- --ignored
```

Which isn't ideal for scripting and is not convenient.

How about?
```
cargo test -- --all
```

Cheers

**Comment by dwijnand** [comment]
(There's a typo in your last suggestion, you mean `cargo test -- --all`, no `run`)

Seeing as `cargo test` just delegates to libtest (rustc's built in unit-test and micro-benchmarking framework) should this issue not be reported upstream to rust-lang/rust (and this issue here labeled [[blocked]](https://github.com/rust-lang/cargo/labels/blocked))?

**Comment by vext01** [comment]
> There's a typo

Thanks, fixed.

> should this issue not be reported upstream

Can do. I didn't know that testing was delegated.

**Comment by dwijnand** [comment]
Me either. I found out while working on https://github.com/rust-lang/cargo/pull/5341.

**Comment by matklad** [comment]
Thanks @dwijnand! 

It is indeed the case that `--ignored` is not handled by Cargo today, so closing this in favor of upstream report!

**Comment by wischi-chr** [comment]
Because I landed here using google and it might help other people (or future me ^^):

`cargo test -- --include-ignored`
